src/components/Events.module.scss

```diff
@@ -7,6 +7,8 @@
   padding-top: 5px;
   padding-left: 10px;
   border-radius: 10px;
+  margin-left: 5%;
```

Comment:
I recommend now going for anything abstract/relevant like this. It can get out of hand when the user's screen is getting bigger. Tbh not even sure how big this is supposed to be. 5% out of what? Width of the element? Width of the full margin?